EveripediaNetwork/ep-ui
# Wrong updated time

# Wrong Updated Time

This PR works on updating the logic for updated time of featured wiki elements

## How should this be tested?

1. Update a featured wiki
2. Navigate to the home page to check the updated time

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2468
